src/main/java/com/github/fukkitmc/fukkit/mixin/net/minecraft/entity/EntityMixin.java

```diff
@@ -643,6 +644,13 @@ public Box fukkit_disableInvalids(Box newBox) {
 		return this.entity;
 	}
 
+	public CraftEntity getBukkitEntity(){
```

Comment:
A little vague if this method is overwritten or new, please add @Override if this method is being ovverriden.

Reply:
Oh I see what he did, he's adding these methods because I named the duck interface method a different name, took me a second